packages/components/src/snackbar/style.scss

```diff
@@ -5,12 +5,7 @@
 	border-radius: $radius-round-rectangle;
 	box-shadow: 0 2px 4px rgba(0, 0, 0, 0.3);
 	color: $white;
-	padding: 15px 10px 15px 20px;
-	width: calc(100% - 20px);
-
-	@include break-small() {
-		width: fit-content;
-	}
```

Comment:
Any reason to remove the "fit-content" ?

This causes this 
<img width="789" alt="Capture d’écran 2019-05-22 à 1 32 41 PM" src="https://user-images.githubusercontent.com/272444/58174610-31907480-7c96-11e9-99ec-ca7159b6cd8d.png">

The first notice takes the width of the second one.

Reply:
I'm mucking around trying to get centred alignments working and getting a bit lost in the multiple different file locations, so there may be some 🚧 in my commits here until it's all ironed out.

My _hope_ is that there'd never be an instance where two snackbar notifications appear at once, so this would be an edge case. (We can add this to the design guidelines.) But in the case that two *do* show up, it's probably better if they're the same width, otherwise it would feel super jarring.